Category.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Category.css">
+    <link rel="stylesheet" href="common.css">
+    <link rel="stylesheet" href="https://canvas-component-lib.netlify.app/Component.css">
+    <title>Document|Category</title>
+</head>
+<body>
+    <div class="desktop-container">
+        <h1>Style Knockout</h1>
+        <div class="Wishlist-container">
+       
+            <!-- Card -->
+            <div class="card2-container wishlist-card">
```

Comment:
A suggestion - Change the class name to something more generic, the wishlist class here doesn't seem fit.
The rest looks good!